app/components/x-root/component.js

```diff
@@ -11,6 +11,13 @@ export default Component.extend({
 	actions: {
 		toggleAside() {
 			this.toggleProperty('layout.showAside')
+		},
+		untoggleAside() {
+			let openMenu = document.getElementsByClassName("show-aside");
```

Comment:
i stedet for at bruge `document` kan du bruge `this.el`. Paa den maade scoper du det til det nuvaerende Ember app.

Reply:
this.el kommer op som undefined.